app/backend/db/models/date_datas.py

```diff
@@ -24,7 +33,7 @@ class DateData(Base):
     date_details = relationship('DateDetails', backref='datas')
     user_id = Column(Integer, ForeignKey('users.id', ondelete='CASCADE'), index=True)
     user = relationship('User', backref='statuses', foreign_keys=[user_id])
-    state = Column(Enum('here', 'not_here', name='answer_state'))
+    state = Column(Enum(Reason.HERE, Reason.NOT_HERE, name='answer_state'))
```

Comment:
You changed the model..
Run alembic to add migration
Make sure it has changes the enum values and default value ( I know it have problem with it)

Reply:
done, didnt need to change anything